XenAdmin/Wizards/XenWizardBase.zh-CN.resx

```diff
@@ -525,175 +170,103 @@
-  </data>
-  <data name="&gt;&gt;panelGeneralInformationMessage.Parent" xml:space="preserve">
-    <value>tableLayoutPanelNavigation</value>
-  </data>
-  <data name="&gt;&gt;panelGeneralInformationMessage.ZOrder" xml:space="preserve">
-    <value>3</value>
-  </data>
-  <data name="tableLayoutPanelNavigation.Dock" type="System.Windows.Forms.DockStyle, System.Windows.Forms">
-    <value>Bottom</value>
-  </data>
-  <data name="tableLayoutPanelNavigation.Location" type="System.Drawing.Point, System.Drawing">
-    <value>0, 465</value>
-  </data>
-  <data name="tableLayoutPanelNavigation.Margin" type="System.Windows.Forms.Padding, System.Windows.Forms">
-    <value>0, 0, 0, 0</value>
-  </data>
-  <data name="tableLayoutPanelNavigation.Padding" type="System.Windows.Forms.Padding, System.Windows.Forms">
-    <value>0, 12, 9, 0</value>
-  </data>
-  <data name="tableLayoutPanelNavigation.RowCount" type="System.Int32, mscorlib">
-    <value>1</value>
-  </data>
-  <data name="tableLayoutPanelNavigation.Size" type="System.Drawing.Size, System.Drawing">
-    <value>809, 37</value>
-  </data>
-  <data name="tableLayoutPanelNavigation.TabIndex" type="System.Int32, mscorlib">
-    <value>1</value>
-  </data>
-  <data name="&gt;&gt;tableLayoutPanelNavigation.Name" xml:space="preserve">
-    <value>tableLayoutPanelNavigation</value>
-  </data>
-  <data name="&gt;&gt;tableLayoutPanelNavigation.Type" xml:space="preserve">
-    <value>System.Windows.Forms.TableLayoutPanel, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
-  </data>
-  <data name="&gt;&gt;tableLayoutPanelNavigation.Parent" xml:space="preserve">
-    <value>$this</value>
-  </data>
-  <data name="&gt;&gt;tableLayoutPanelNavigation.ZOrder" xml:space="preserve">
-    <value>1</value>
+  <data name="labelGeneralInformationMessage.Text" xml:space="preserve">
+    <value>仅许可此池使用 XenApp/XenDesktop 工作负载</value>
   </data>
-  <data name="tableLayoutPanelNavigation.LayoutSettings" type="System.Windows.Forms.TableLayoutSettings, System.Windows.Forms">
-    <value>&lt;?xml version="1.0" encoding="utf-16"?>&lt;TableLayoutSettings>&lt;Controls>&lt;Control Name="buttonPrevious" Row="0" RowSpan="1" Column="2" ColumnSpan="1" />&lt;Control Name="buttonNext" Row="0" RowSpan="1" Column="3" ColumnSpan="1" />&lt;Control Name="buttonCancel" Row="0" RowSpan="1" Column="4" ColumnSpan="1" />&lt;Control Name="panelGeneralInformationMessage" Row="0" RowSpan="1" Column="0" ColumnSpan="2" />&lt;/Controls>&lt;Columns Styles="Percent,100,AutoSize,0,AutoSize,0,AutoSize,0,AutoSize,0" />&lt;Rows Styles="AutoSize,0" />&lt;/TableLayoutSettings></value>
+  <data name="XSHelpButton.Image" type="System.Drawing.Bitmap, System.Drawing" mimetype="application/x-microsoft.net.object.bytearray.base64">
+    <value>
+        iVBORw0KGgoAAAANSUhEUgAAABkAAAAYCAYAAAAPtVbGAAAABGdBTUEAALGOfPtRkwAAACBjSFJNAACH

```

Comment:
Please remove the Image for the XSHelpButton from the resx file, as the image is assigned at run-time, when the HelpButton is initialised.